sdk/src/main/java/io/dapr/utils/DurationUtils.java

```diff
@@ -65,8 +89,7 @@ public static String convertDurationToDaprFormat(Duration value) {
 
     // return empty string for anything negative, it'll only happen for reminder "periods", not dueTimes.  A
     // negative "period" means fire once only.
-    if (value == Duration.ZERO
-        || (value.compareTo(Duration.ZERO) == 1)) {
```

Comment:
`compareTo` can produce any value greater than 1, by contract any positive value means the value is greater. This failed the test when only milliseconds were given ...